easypost/requestor.py

```diff
@@ -131,21 +149,23 @@ def requests_request(self, method, abs_url, headers, params):
             )
         return http_body, http_status
 
-    def urlfetch_request(self, method, abs_url, headers, params):
+    def urlfetch_request(
+        self, method: str, abs_url: str, headers: Dict[str, Any], params: Dict[str, Any]
+    ) -> Tuple[str, int]:
         """Make a request by using the `urlfetch` library."""
         fetch_args = {"method": method, "headers": headers, "validate_certificate": False, "deadline": timeout}
-        if method == "post" or method == "put":
+        if method == "get" or method == "delete":
+            fetch_args["url"] = self.add_params_to_url(url=abs_url, params=params)
+        elif method == "post" or method == "put":
             fetch_args["url"] = abs_url
             fetch_args["payload"] = json.dumps(params, default=self._utf8)
-        elif method == "get" or method == "delete":
-            fetch_args["url"] = self.add_params_to_url(abs_url, params)
         else:
             raise Error(f"Bug discovered: invalid request method: {method}. Please report to {SUPPORT_EMAIL}.")
 
         try:
-            from google.appengine.api import urlfetch
+            from google.appengine.api import urlfetch  # type: ignore
 
-            result = urlfetch.fetch(**fetch_args)
+            result = urlfetch.fetch(**fetch_args)  # type: ignore
```

Comment:
The `#type: ignore` is an indicator to `mypy` to ignore type-hinting issues here. Otherwise, this line is a nightmare because it could realistically be several different types and that spreads up the chain to the rest of the app.